src/main/java/io/apisense/embed/influx/configuration/server/HttpConfigurationSection.java

```diff
@@ -1,8 +1,11 @@
 package io.apisense.embed.influx.configuration.server;
 
 public class HttpConfigurationSection extends AbstractConfigurationSection {
-    public HttpConfigurationSection(int port) {
+    public HttpConfigurationSection(int port, boolean auth) {
         super("http");
         addProperty(ConfigurationProperty.BIND_ADDRESS, ":" + port);
+        if (auth) {
```

Comment:
The condition here is not necessary, you can always put the `auth` value for the property `AUTH_ENABLED`.